lib/tasks/gather.js

```diff
@@ -300,27 +301,27 @@ class TaskGather extends SttTask {
     const opts = this.setChannelVarsForStt(this, this.sttCredentials, this.data.recognizer);
     switch (this.vendor) {
       case 'google':
-        this.bugname = 'google_transcribe';
+        this.bugname = `${this.bugname_prefix}google_transcribe`;
```

Comment:
are we guaranteed to always have `bugname_prefix` property of this?  If not we would we have "undefined_google_transcribe"?  If so I would rather have `${this.bugname || ''}google_transcribe`

Reply:
By default is empty string:
https://github.com/jambonz/jambonz-feature-server/pull/576/files#diff-540439258e43bbda786b9a2d537d8b5c87ec55fc1f92c69c291861368f5fa94dR53